src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/SubModel.h

```diff
@@ -147,24 +148,70 @@ class SubModel
 
     /**
      * @brief Access an element of the force/torque sensor list.
+     * @param index is the index of the force/torque sensor in the submodel
      * @return FT object associated with the specified index.
      */
     const FT& getFTSensor(const int index) const;
 
+    /**
+     * @brief Access an element of the force/torque sensor list.
+     * @param is the name of the force/torque sensor.
+     * @return FT object associated with the specified name.
+     */
+    const FT& getFTSensor(const std::string name) const;
```

Comment:
I would suggest passing by reference. This hold also for all the methods below where you pass a ``const std::string`` instead of a ``const std::string&``